### ISSUE TITLE
OpenAI integration

### ISSUE THREAD

**Issue by lake4790k** [Kaixhin/Atari#13]
Would be interesting to be able to run Atari against the [OpenAI  gym](https://github.com/openai/gym) environment. Currently OpenAI is in python, maybe would be possible to have a [pytorch](https://github.com/hughperkins/pytorch) runner in Atari that integrates the two. Or also there could be general torch support for OpenAI, ie. [this issue](https://github.com/openai/gym/issues/12) with [fb-python](https://github.com/facebook/fblualib/blob/master/fblualib/python/README.md).

(btw with the Deepmind capitulation things are going towards python/tf, not that I would follow at this point...)

**Comment by Kaixhin** [comment]
Yep would definitely love to hook up to the OpenAI Gym API, but would leave this until we hear more about properly supported integration for Torch (i.e. when we hear about progress from Soumith).

**Edit 2016-06-25:** Possible integration through [HTTP](https://github.com/openai/gym-http-api).
